app/views/comments/_form.html.erb

```diff
@@ -1,4 +1,5 @@
 <%= form_with(model: [ @article, @article.comments.build ], local: true) do |form| %>
+
   <p>
     <%= form.label :commenter %><br>

```

Comment:
It's my impression that using `<br>` tags is not a best practice, I need to research it more though.  
The alternative is to position the `label`s in relation to the `input`s using CSS.  
Leave it for now, it's just an FYI.